ntbs-service/wwwroot/source/app.ts

```diff
@@ -39,8 +41,12 @@ import ConfirmComponent from "./Components/ConfirmComponent";
 // For compatibility with IE11
 require("es6-promise").polyfill();
 
-// Vue needs to be the first thing to load!
-// Otherwise, it replaces the templates of its components with fresh content, potentially overwriting changes from other scripts!
+if (config.env === "production") {
+    Sentry.init({
+        dsn: 'https://83b245064a684fa7ac86658bf38d2ad3@sentry.io/2862017',
```

Comment:
would be nice to have a comment explaining where this numbers/link coming from

Reply:
Yeah, that makes sense!